File: benchmark/cases/hash_map/hash_map.js
const map = new Map();
for (let i = 0; i < 100000; i++) {
    map.set(`key_${i}`, i);
}
let sum = 0;
for (let i = 0; i < 100000; i++) {
    sum += map.get(`key_${i}`);
}
console.log(sum);

File: benchmark/cases/hash_map/hash_map.rs
use std::collections::HashMap;

fn main() {
    let mut map = HashMap::new();
    for i in 0..100_000i64 {
        map.insert(format!("key_{}", i), i);
    }
    let mut sum: i64 = 0;
    for i in 0..100_000i64 {
        sum += map[&format!("key_{}", i)];
    }
    println!("{}", sum);
}
